ci/test/format_no_generated_java_test.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Does this script have the execute bit set?

Reply:
Yup, I just typed it wrong in the CI configuration. PTAL.